src/stim/stabilizers/tableau.pybind.cc

```diff
@@ -208,6 +208,34 @@ void stim_pybind::pybind_tableau(pybind11::module &m) {
         )DOC")
             .data());
 
+    c.def(
+        "to_pauli_string",
+        [](const Tableau &self) {
+            return PyPauliString(self.to_pauli_string());
+        },
+        clean_doc_string(u8R"DOC(
+            If the tableau fixes each pauli upto sign, creates the pauli
```

Comment:
Do a single-line summary, then elaborate in a new paragraph. E.g.

```
Returns a Pauli string equivalent to the tableau.

If the tableau isn't equivalent to a pauli string, an error is raised.
```

Reply:
:+1: